db/migrate/20250706230741_change_flipper_gates_value_to_text.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ChangeFlipperGatesValueToText < ActiveRecord::Migration[7.2]
+  def up
+    # Ensure this incremental update migration is idempotent
+    return unless connection.column_exists? :flipper_gates, :value, :string
+
+    if index_exists? :flipper_gates, [:feature_key, :key, :value]
+      remove_index :flipper_gates, [:feature_key, :key, :value]
+    end
+    change_column :flipper_gates, :value, :text
+    safety_assured do
+      add_index :flipper_gates, [:feature_key, :key, :value], unique: true, length: { value: 255 }
+    end
```

Comment:
I'm not sure if how big is `flipper_gates`, if it is I'll move it to new migration that creates the index concurrently.